S17-procasync/stress.t

```diff
@@ -93,3 +93,21 @@ if run :!out, :!err, «perl -e 'print 42'» {
 else {
     skip 'need `perl` to run this test';
 }
+
+{
+  my $prog   = $*DISTRO.is-win ?? 'cmd'   !! 'cat';
+  my @target = $*DISTRO.is-win ?? «/c ""» !! '/dev/null';
+
+  for ^1200 {
+    print "\r$_ ";
```

Comment:
Tests should output nothing but TAP. This `print` must no be in production.

Reply:
fixed